### PR TITLE
LRQA-70420, LRQA-70423, LRQA-70416, LRQA-70422 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/LRQA-70420
https://issues.liferay.com/browse/LRQA-70423
https://issues.liferay.com/browse/LRQA-70416
https://issues.liferay.com/browse/LRQA-70422

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:sf

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Hi @Kvale1733 ,
Please fix the typo in the commit's title of 0da8757a49d1375a84b506cd08a45d6d9d6e67d4
The code looks good to me.

**Comment by Kvale1733** [comment]
fixed title

**Comment by Kvale1733** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase [hunk 1]
```diff
@@ -83,7 +83,7 @@ definition {
 	@description = "This is a use case for LRQA-42712. Verify guest users cannot configure public pages."
 	@priority = "4"
 	@uitest
-	test ViewNoDisplayPortletConfigurationPermission {
+	test ViewNoDisplayPortletConfigurationPermissionAsGuest {
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase:66]: The other test case which you rewrote is similiar in description and in goal too. For these reasons and in the aspect of consistency I would suggest using similiar naming like on the other case (It was "**CannotConfigureBlogsPortletAsGuestPG**")

Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase:66]: I agree

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase [hunk 2]
```diff
@@ -88,9 +68,8 @@ definition {
 		property test.assert.javascript.errors = "false";
 		property test.assert.warning.exceptions = "true";
 
-		var portalURL = PropsUtil.get("portal.url");
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase:89]: If it happens in a MACRO file, then I agree with you @weller0246, but in this test case level, I also don't see a point to pick the portal URL from property first, then fill it to URL variables.
I think we could keep this change.

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Hi @Kvale1733, could you please change the test case name as @weller0246 mentioned?

**Comment by Kvale1733** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase (hunk 1 above)
Comment (Kvale1733) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase]: Done!

**Review by weller0246 — APPROVED** [review]
LGTM

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase [hunk 3]
```diff
@@ -43,52 +44,33 @@ definition {
 		task ("Add new blog") {
 			var browserType = PropsUtil.get("browser.type");
```
Comment (ferenconodiD) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase:45]: Line can be deleted as _browserType_ variable is no longer used!

**Review by ferenconodiD — CHANGES_REQUESTED** [review]
@Kvale1733 Please resolve review remarks!

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**